test/vast_tracker.js

```diff
@@ -424,6 +424,26 @@ describe('VASTTracker', function() {
           _eventsSent[1].should.eql('clickthrough');
         });
       });
+
+      describe('#click with fallback', () => {
+        let fallbackClickThroughURL =
+          'http://example.com/fallback-clickthrough';
+        before(done => {
+          // Init tracker with empty data (we don't need any since we'll test a fallback)
+          this.Tracker = new VASTTracker(vastClient, {}, {});
```

Comment:
we can do `const tracker = new VASTTracker(vastClient, {}, {});`. IDK why we were storing the tracker instance as a property of the test, but this is not a good idea since it may affect other unit tests (and it does, see the changes you had to make on line 496 😉)

Ideally every describe should have his own local instance of the tracker